### PR TITLE
feat: Adding Basildon Council

### PR DESCRIPTION
**Pull request by eugeneniemand**
Adding Basildon Council (Essex)

### CODE REVIEW

File: uk_bin_collection/tests/input.json [hunk 1]
```diff
@@ -42,6 +42,12 @@
         "wiki_name": "Barnsley Metropolitan Borough Council",
         "wiki_note": "To get the UPRN, you will need to use [FindMyAddress](https://www.findmyaddress.co.uk/search)."
     },
+"BasildonCouncil": {
+        "skip_get_url": true,
+        "uprn": "10013350430",
+        "url": "https://basildonportal.azurewebsites.net/api/getPropertyRefuseInformation",
+        "wiki_name": "Basildon Council"
+    },
     "BasingstokeCouncil": {
```
Comment (eugeneniemand) [inline uk_bin_collection/tests/input.json]:
```suggestion
        "BasingstokeCouncil": {
```

**Comment by robbrad** [comment]
Looks good @eugeneniemand 

Check https://github.com/robbrad/UKBinCollectionData/actions/runs/11182453170/job/31090642306#step:12:1

The date format needs to match the schema. Reason for this is consistency with the custom component 

Thanks
Rob

**Comment by eugeneniemand** [comment]
@robbrad Apologies mate, I dont know python that well, and knocked this together in about 20 minutes never even looking at the date format as I'm so used to using the ISO 8601 format that is didnt occur to me that it needs to be changed. Should all be fixed now

**Comment by eugeneniemand** [comment]
@robbrad not sure what to do about commit lint error, I see many other failures so not sure if its required

**Review by robbrad — APPROVED** [review]

**Comment by eugeneniemand** [comment]
@robbrad just curious if this has gone into release 0.92.0 as I cant see it when adding the integration in HA? Did I miss something to enable it?

**Comment by robbrad** [comment]
Yeh unfortunately you need to prefix all commit message with either feat: or fix:

Otherwise the pipeline won't pick up the version release. It's mentioned in the contributing.md

Unfortunately it's quite hard to rework multiple commits so I tend to merge peoples work if it's passing the tests which yours was.

Options are wait until someone else does the next commit or make a small commit and I'll approve

**Comment by robbrad** [comment]
@eugeneniemand fixing it now

**Comment by robbrad** [comment]
Ok v93 should be good

**Pull request merged**